.github/workflows/ci.yaml

```diff
@@ -15,14 +15,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tox-env: [py36, py37, py38, docs, linters, packaging]
+        tox-env: [py36, py37, py38, py39, docs, linters, packaging]
```

Comment:
Also, support 3.10, and remove support for 3.6 as it reached EOL.

And don't forget to add a change log for new/removed Python versions.

Reply:
And don't forget to update versions here:
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/setup.py#L47
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/setup.py#L81-L83
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/tox.ini#L2
- https://github.com/genialis/resolwe-bio-py/blob/e4f6935f509bb1f935abef5414d745b4c16939ef/tox.ini#L29

Reply:
Done.

Reply:
The problem is in genialis-base settings for resdk e2e tests which are used.
See https://github.com/genialis/genialis-base/pull/689

Reply:
Please also change `README.rst` and `docs/contributing.rst` to reflect the removal of support for `Python 3.6`.